src/docs/sdk/event-payloads/contexts.mdx

```diff
@@ -375,3 +375,26 @@ The following example illustrates the contexts part of the <Link to="/sdk/event-
   }
 }
 ```
+
+## Current Culture Context
```

Comment:
This "current culture" is a .NET thing because you can have a different culture set on a thread. So the device culture can be one, and the thread another. We could simplify here (and/or reword it) if that makes sense.

Reply:
@bruno-garcia would be possible then that you are sending 2+ current_culture values? per thread for example?

Reply:
.NET currently sends two (if they differ). `Culture` and `Culture Culture`. With this format as key :hides:

But if this lands in the spec we'd be adjusting accordingly

Reply:
Contexts per thread is a concept we don't really support, but you can send several contexts with different keys.